source/scripts/model/Monster.js

```diff
@@ -100,6 +100,49 @@ export default class Monster {
         this.position.x += movement.x
         this.position.y += movement.y
     }
+    pursuit(magnitude) {
```

Comment:
It should return an object literal as a movement, as per the expected format of {x: #, y: #}